library/coverage/result_spec.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+config_file = File.expand_path('../fixtures/start_coverage.rb', __FILE__)
+
+describe "Coverage.result" do
+  it "needs to be reviewed for spec completeness"
```

Comment:
This can be removed as soon as there are specs, even if they are not exhaustive.